frontend/common/columns.tsx

```diff
@@ -271,3 +286,37 @@ export function useTypeColumn<T extends object>(options: {
   );
   return column;
 }
+export function useScopeColumn<T extends { scope?: string }>(options?: {
```

Comment:
I believe this column, and the `useExpires` column only exist in the tokens list...they are one-off columns.  In the interest of WET (write everything twice) before abstracting it out I believe we should move these to live inside the useTokensColumns file.

Reply:
I contemplated putting them in the useTokensColumns file but I saw that this page will need it too https://awx-devel.aap-ci-k3s-01.gcp.testing.ansible.com/#/users/3/tokens